apps/web/modules/settings/billing/components/BillingCredits.tsx

```diff
@@ -132,117 +169,154 @@ export default function BillingCredits() {
-      ? (creditsData.credits.totalRemainingMonthlyCredits / creditsData.credits.totalMonthlyCredits) * 100
-      : 0;
+  const totalCredits =
+    (creditsData.credits.totalCreditsForMonth ?? 0) ||
+    creditsData.credits.totalMonthlyCredits + creditsData.credits.additionalCredits;
+  const totalUsed =
+    (creditsData.credits.totalCreditsUsedThisMonth ?? 0) ||
+    totalCredits - (creditsData.credits.totalRemainingCreditsForMonth ?? 0);
+
+  const teamCreditsPercentageUsed = totalCredits > 0 ? (totalUsed / totalCredits) * 100 : 0;
+  const numberFormatter = new Intl.NumberFormat();
 
   return (
-    <div className="border-subtle mt-8 space-y-6 rounded-lg border px-6 py-6 pb-6 text-sm sm:space-y-8">
-      <div>
-        <h2 className="text-base font-semibold">{t("credits")}</h2>
-        <ServerTrans
-          t={t}
-          i18nKey="view_and_manage_credits_description"
-          components={[
-            <Link
-              key="Credit System"
-              className="underline underline-offset-2"
-              target="_blank"
-              href="https://cal.com/help/billing-and-usage/messaging-credits">
-              Learn more
-            </Link>,
-          ]}
-        />
-        <div className="-mx-6 mt-6">
-          <hr className="border-subtle" />
+    <>
+      <div className="bg-muted border-muted mt-5 rounded-xl border p-1">
+        <div className="flex flex-col gap-1 px-4 py-5">
+          <h2 className="text-default text-base font-semibold leading-none">{t("credits")}</h2>
+          <p className="text-subtle text-sm font-medium leading-tight">{t("view_and_manage_credits")}</p>
         </div>
-        <div className="mt-6">
-          {creditsData.credits.totalMonthlyCredits > 0 ? (
-            <div className="mb-4">
-              <Label>{t("monthly_credits")}</Label>
-              <ProgressBar
-                color="green"
-                percentageValue={teamCreditsPercentageUsed}
-                label={`${Math.max(0, Math.round(teamCreditsPercentageUsed))}%`}
-              />
-              <div className="text-subtle">
-                <div>
-                  {t("total_credits", {
-                    totalCredits: creditsData.credits.totalMonthlyCredits,
-                  })}
+        <div className="bg-default border-muted flex w-full rounded-[10px] border px-5 py-4">
+          <div className="w-full">
+            {totalCredits > 0 ? (
+              <>
+                <div className="mb-4">
+                  <CreditRow
+                    label={t("monthly_credits")}
+                    value={creditsData.credits.totalMonthlyCredits ?? 0}
+                    isBold={true}
+                    underline="dashed"
+                  />
+                  <CreditRow label={t("credits_used")} value={totalUsed} underline="solid" />
+                  <CreditRow
+                    label={t("total_credits_remaining")}
+                    value={creditsData.credits.totalRemainingCreditsForMonth}
```

Comment:
This should only be monthly credits not monthly + additional

<img width="857" height="250" alt="Screenshot 2025-09-19 at 11 02 06 AM" src="https://github.com/user-attachments/assets/ab0fcf66-5793-4f37-a2d4-18440c4a05b2" />